### DISCUSSION TITLE
Posting job offers

### DISCUSSION THREAD

**Discussion by indominus-hub** [Kitware/trame#53] [General]
Hi guys,

Is it possible to share job offers here?

I am interested in hiring a trame expert for my company, but I don't wwant to disturb this nice community.

So please, moderators/administrators...let me know.

Kind regards,

Isma

**Comment by jourdain** [comment]
Hi Isma, 

That is a good question and thanks for thinking of the community first. 
Let us think about it and we'll get back on that thread with a better idea on how we want to proceed. 

In general, even if Kitware built those tools and have support contracts available, we try to help the community as much as we could first and then fall back to a contract suggestion when questions/help start to be too involved. And mostly that is to be respectful of our community.
But maybe having a dedicated thread/category for jobs could be useful to the community and not too intrusive?

Any comment/idea/suggestion from the community?

Thanks again for your interest in trame!

Seb

**Comment by jourdain** [comment]
I've just created a [new category](https://github.com/Kitware/trame/discussions/categories/jobs-funded-requests). It should be use for any job offer/dev service around trame and/or task request with funding support.

**Reply by indominus-hub** [reply]
Than you very much @jourdain 

Definitely this will help! 

I will post the jib offer in the new category in a while.

All the best
